android/jni/com/mapswithme/maps/editor/Editor.cpp

```diff
@@ -185,13 +185,14 @@ Java_com_mapswithme_maps_editor_Editor_nativeSetDefaultName(JNIEnv * env, jclass
 JNIEXPORT jstring JNICALL
 Java_com_mapswithme_maps_editor_Editor_nativeGetStreet(JNIEnv * env, jclass)
 {
-  return jni::ToJavaString(env, g_editableMapObject.GetStreet());
+  // TODO(yunikkk) use "GetStreet().second" to get localized street
+  return jni::ToJavaString(env, g_editableMapObject.GetStreet().m_defaultName);
 }
 
 JNIEXPORT void JNICALL
 Java_com_mapswithme_maps_editor_Editor_nativeSetStreet(JNIEnv * env, jclass, jstring street)
 {
-  g_editableMapObject.SetStreet(jni::ToNativeString(env, street));
+  g_editableMapObject.SetStreet({jni::ToNativeString(env, street), ""});
```

Comment:
Не консистентно - код в ядре для отображения устанавливает обе улицы, а код из UI, при выборе из списка, перевод будет сбрасывать?

> On Apr 8, 2016, at 23:48, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/maps/editor/Editor.cpp <https://github.com/mapsme/omim/pull/2850#discussion_r59087107>:
> 
> >  }
> >  
> >  JNIEXPORT void JNICALL
> >  Java_com_mapswithme_maps_editor_Editor_nativeSetStreet(JNIEnv * env, jclass, jstring street)
> >  {
> > -  g_editableMapObject.SetStreet(jni::ToNativeString(env, street));
> > +  g_editableMapObject.SetStreet({jni::ToNativeString(env, street), ""});
> SetStreet(string const & defaultName, string const & localizedName = "")
> ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2850/files/040c28c51ccfe28842f35433adac33c8ce84fb66#r59087107>